modules.d/95znet/parse-ccw.sh

```diff
@@ -4,4 +4,24 @@ for ccw_arg in $(getargs rd.ccw -d 'rd_CCW=') $(getargs rd.znet -d 'rd_ZNET=');
     echo $ccw_arg >> /etc/ccw.conf
 done
 
+for ifname in $(getargs rd.znet_ifname); do
+    echo "$ifname" | {
```

Comment:
I don't think that the pipe to the whole block is necessary here. I would suggest doing
IFS=: read ifname_if ifname_subchannels _ <<< "$ifname"

Reply:
I've no problem to change using read <<< $var style, will update. btw is that a preferred code style or have some advantage?

Reply:
I think it is used in other places, but personally I find it a bit easier to read.